ListItemFormatter.podspec

```diff
@@ -10,11 +10,11 @@ Pod::Spec.new do |s|
   s.social_media_url = 'https://twitter.com/liamnichols_'
   s.source = { :git => 'https://github.com/liamnichols/ListItemFormatter.git', :tag => "v#{s.version}" }
 
-  s.ios.deployment_target = '9.0'
+  s.ios.deployment_target = '11.0'
```

Comment:
What’s the reason for changing this to iOS 11?

Reply:
I was wondering to keep up minimum deployment target to the project, so that it will unveil some hidden warnings.  If it is not necessary, I will change it back to `9.0`

Reply:
Hmm good point... I think that since the code is compatible with iOS 9 and while there is currently no motivation/gain from dropping iOS 9/10 support then it makes sense to keep it that way 

In terms of soft-deprecation warnings, I wonder if there is another way to spot them 🤔 Feel free to open a separate issue if you’d like and then we can see what other projects do as a starting point maybe

Reply:
Great, I did roll back to iOS 9 as well as changed to project level control Swift version, thanks fo the tips